spec/factories/task_managements.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :task_management do
+    task_id 1
+    tasker_id 1
+    taskee_id 2
+    task_desc "A task"
```

Comment:
Consider using https://github.com/stympy/faker#fakerlorem for the description and other generic attributes.